modules/test/playwright/pages/style-book-web/StyleBookPage.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page} from '@playwright/test';
+
+import {ProductMenuPage} from '../product-navigation-product-menu/ProductMenu.page';
+
+export class StyleBookPage {
+	readonly productMenuPage: ProductMenuPage;
+	readonly page: Page;
+
+	constructor(page: Page) {
+		this.productMenuPage = new ProductMenuPage(page);
+		this.page = page;
+	}
+
+	async goto() {
+		await this.productMenuPage.goToStyleBooksMenuItem();
+	}
+
+	async createStyleBook(styleBookName: string) {
+		await this.goto();
+
+		await this.page.click(
+			`[data-qa-id="creationMenuNewButton"]:has-text("New")`
```

Comment:
We should be using `getByTestId` instead of directly checking data-qa-id. Make sure to update all instances.